orttraining/orttraining/test/graph/training_session_test.cc

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Consider putting this file under orttraining/orttraining/test/session so that it's under similar path as orttraining/orttraining/session/training_session.

Reply:
done